tsdb/chunks/head_chunks_test.go

```diff
@@ -231,15 +237,18 @@ func TestChunkDiskMapper_Truncate(t *testing.T) {
 
 	// Truncating files after restart.
 	require.NoError(t, hrw.Truncate(sixthFileMinT))
-	verifyFiles([]int{6, 7, 8, 9, 10})
 
-	// As the last file was empty, this creates no new files.
-	require.NoError(t, hrw.Truncate(sixthFileMinT+1))
-	verifyFiles([]int{6, 7, 8, 9, 10})
+	// Add a chunk to trigger truncation.
```

Comment:
```suggestion
	// Add a chunk to trigger cutting of new file.
```

Reply:
updated in 4d3d59494f79d57c0faa143668544adbedc81f97